src/Microsoft.AspNetCore.Mvc.Analyzers/DiagnosticDescriptors.cs

```diff
@@ -7,40 +7,22 @@ namespace Microsoft.AspNetCore.Mvc.Analyzers
 {
     public static class DiagnosticDescriptors
     {
-        public static readonly DiagnosticDescriptor MVC1000_ApiActionsMustBeAttributeRouted =
+        public static readonly DiagnosticDescriptor MVC1000_HtmlHelperPartialShouldBeAvoided =
```

Comment:
I could be the same diagnostic just with a parameter for the method name,

Reply:
Yeah, that should be do-able